### PR TITLE
fix mypy lnurldevices

### PR DESCRIPTION
**Pull request by dni**

### CODE REVIEW

File: lnbits/extensions/lnurldevice/views.py [hunk 1]
```diff
@@ -62,4 +57,6 @@ async def img(request: Request, lnurldevice_id):
         raise HTTPException(
             status_code=HTTPStatus.NOT_FOUND, detail="LNURLDevice does not exist."
         )
-    return lnurldevice.lnurl(request)
+    # error: "lnurldevices" has no attribute "lnurl"
+    # return lnurldevice.lnurl(request)
```
Comment (dni) [inline lnbits/extensions/lnurldevice/views.py:61]: @arcbtc did you forget this bit when refactoring?

**Comment by callebtc** [comment]
Has conflicts :) Is it ready for merge other than that?

**Comment by dni** [comment]
going solve conflicts now, but @arcbtc  has to test this one

**Comment by arcbtc** [comment]
Yah ist gud! All tested, working great LGTM

**Pull request merged**